package.json

```diff
@@ -14,33 +14,27 @@
-    "check-intl-merge:web": "turbo run check-intl --filter=@gc-digital-talent/i18n -- --dir $INIT_CWD --en ./apps/web/src/lang/en.json --fr ./apps/web/src/lang/fr.json --rm-orphaned --output-untranslated ./apps/web/src/lang/untranslated.json --whitelist ./apps/web/src/lang/whitelist.yml --merge-fr ./apps/web/src/lang/newTranslations.json",
+    "codegen": "turbo run codegen --filter @gc-digital-talent/graphql",
     "clean": "turbo run clean",
+    "dev": "h2-build && turbo run dev --filter @gc-digital-talent/web",
+    "dev:fresh": "h2-build && turbo run dev --filter @gc-digital-talent/web --force",
+    "check-intl:i18n": "turbo run check-intl --filter @gc-digital-talent/i18n -- --dir $INIT_CWD --en ./packages/i18n/src/lang/en.json --fr ./packages/i18n/src/lang/fr.json --rm-orphaned --output-untranslated ./packages/i18n/src/lang/untranslated.json --whitelist ./packages/i18n/src/lang/whitelist.yml",
+    "check-intl:web": "turbo run check-intl --filter @gc-digital-talent/i18n -- --dir $INIT_CWD --en ./apps/web/src/lang/en.json --fr ./apps/web/src/lang/fr.json --rm-orphaned --output-untranslated ./apps/web/src/lang/untranslated.json --whitelist ./apps/web/src/lang/whitelist.yml",
+    "check-intl-merge:i18n": "turbo run check-intl --filter @gc-digital-talent/i18n -- --dir $INIT_CWD --en ./packages/i18n/src/lang/en.json --fr ./packages/i18n/src/lang/fr.json --rm-orphaned --output-untranslated ./packages/i18n/src/lang/untranslated.json --whitelist ./packages/i18n/src/lang/whitelist.yml --merge-fr ./packages/i18n/src/lang/newTranslations.json",
+    "check-intl-merge:web": "turbo run check-intl --filter @gc-digital-talent/i18n -- --dir $INIT_CWD --en ./apps/web/src/lang/en.json --fr ./apps/web/src/lang/fr.json --rm-orphaned --output-untranslated ./apps/web/src/lang/untranslated.json --whitelist ./apps/web/src/lang/whitelist.yml --merge-fr ./apps/web/src/lang/newTranslations.json",
+    "check-intl": "pnpm run intl-extract & pnpm run check-intl:i18n & pnpm run check-intl:web",
     "intl-extract": "turbo run intl-extract",
     "intl-compile": "turbo run intl-compile",
-    "dev": "h2-build && turbo run dev --filter='./apps/*' --filter='!@gc-digital-talent/e2e'",
-    "dev:fresh": "h2-build && turbo run dev --filter='./apps/*' --filter='!@gc-digital-talent/e2e' --force",
-    "dev:project": "turbo run dev",
     "lint": "turbo run lint",
-    "storybook:project": "pnpm run storybook",
     "storybook": "pnpm --filter @gc-digital-talent/web run storybook",
     "storybook:design-system": "pnpm --filter @gc-digital-talent/web run storybook:design-system",
     "storybook:web": "pnpm --filter @gc-digital-talent/web run storybook:web",
     "build-storybook": "pnpm --filter @gc-digital-talent/web run build-storybook",
     "test": "turbo run test",
-    "watch": "turbo run watch --concurrency=30",
-    "watch:project": "turbo run watch",
+    "watch": "turbo run watch --filter @gc-digital-talent/web --filter @gc-digital-talent/graphql",
```

Comment:
[Similar](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9874#discussion_r1535679122) to @petertgiles's comment, `watch` works but is very slow (10+ seconds) for webpack to compile when changing anything outside `apps/web`: is this different than before?

Reply:
Yeah, I just checked main and it takes roughly the same amount of time give or take. Only more reason to move on to vite :rofl: